### ISSUE TITLE
BUG: radio buttons chioce type returns an array

### ISSUE THREAD

**Issue by nioxiao** [kristijanhusak/laravel-form-builder#686]
```php
$this->add('yes_or_no', 'choice', [
    'label' => 'Yes or No：',
    'choices' => ['0' => 'No', '1' => 'Yes'],
    'choice_options' => [
        'wrapper' => ['class' => 'choice-wrapper'],
        'label_attr' => ['class' => 'label-class'],
    ],
    'selected' => '0',
    'expanded' => true,
    'multiple' => false
]);
```
$form->getFieldValues() returns ```array('yes_or_no' => array('0'))``` after submitted. It should be ```array('yes_or_no' => '0')```

**Comment by rjp-thijs** [comment]
Appears to be caused by https://github.com/kristijanhusak/laravel-form-builder/issues/675 / https://github.com/kristijanhusak/laravel-form-builder/pull/676

**Comment by rudiedirkx** [comment]
How you do make the entire form? Is `yes_or_no` inside a form inside a collection?

**Comment by rudiedirkx** [comment]
Crap. I see it. `ParentType`. I #&$_@ hate that sneaky `ParentType`! I never use `choice` because it's so weird.

**Comment by rudiedirkx** [comment]
I've released [`1.51.1`](https://packagist.org/packages/kris/laravel-form-builder#1.51.1). My apologies.

**Comment by rjp-thijs** [comment]
Thankyou!
